recipes/discord/webview.js

```diff
@@ -33,9 +33,11 @@ module.exports = (Ferdium, settings) => {
 
     if (link || button) {
       const url = link ? link.getAttribute('href') : button.getAttribute('title');
-      const stayInsideDiscord = url.includes('https://discordapp.com/channels/');
+      const stayInsideDiscord =
+        url.includes('https://discordapp.com/channels/') ||
+        url.includes('https://discord.com/channels/');
 
-      if (!Ferdium.isImage(url) && !stayInsideDiscord) {
+      if (!Ferdium.isImage(link) && !stayInsideDiscord) {
```

Comment:
What happens if the user clicked on a button (and the `link` variable is undefined)?

Reply:
Inside Ferdium.IsImage we make a check and return false if "link" is undefined